src/daemon/index.js

```diff
@@ -63,7 +64,7 @@ module.exports = async function (ctx) {
       }
 
       log.end()
-      updateStatus(STATUS.STARTING_FINISHED)
+      updateStatus(STATUS.STARTING_FINISHED, id)
```

Comment:
We're not scraping the logs anymore, so we send the peer id on the event once the startup is finished.